src/test/storage/chunk_encoder_test.cpp

```diff
@@ -146,4 +151,26 @@ TEST_F(ChunkEncoderTest, EncodeMultipleChunksUsingSameEncoding) {
   verify_encoding(_table->get_chunk(ChunkID{1u}), unencoded_chunk_spec);
 }
 
+TEST_F(ChunkEncoderTest, ReencodingTable) {
+  const auto chunk_encoding_specs =
```

Comment:
Comment on what this holds?

Reply:
Added `// Encoding specifications which will be applied one after another to the chunk.`